### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by e111077**
DO NOT MERGE

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -17,24 +17,57 @@
...
+        "platinum-sw": "polymerelements/platinum-sw#^1.0.0",
+        "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
+        "note-app-elements": "polymerlabs/note-app-elements",
+        "test-fixture": "polymerelements/test-fixture#^1.0.0",
+        "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
+        "web-component-tester": "f27c6e0b11e476b48cdeb06e3d494ea0f132e503"
+      }
+    }
+  },
+  "resolutions": {
```
Comment (notwaldorf) [inline bower.json]: Are these still needed?

File: test/index.html [hunk 2]
```diff
@@ -18,7 +18,8 @@
 <body>
   <script>
     WCT.loadSuites([
-      './app-pouchdb-document.html'
+      './app-pouchdb-document.html?dom=shadow',
+      './app-pouchdb-document.html?wc-shadowdom=true&wc-register=true'
```
Comment (notwaldorf) [inline test/index.html]: wc-ce 🎏

**Comment by notwaldorf** [comment]
@e111077 omg i can't turn my green check into a red x but like do the needful yo

File: bower.json (hunk 1 above)
Comment (e111077) [inline bower.json]: thanks for reminding me to revert the pointer of wct to the RC and 4.0 respectively

**Comment by e111077** [comment]
@notwaldorf I did the needs. Can you please mark in the tracker whether it's a okayed?

**Pull request merged**